app/models/Service.ts

```diff
@@ -0,0 +1,93 @@
+import { get } from '../utils/DataService';
+import { parseAPISchedule } from '../utils/transformSchedule';
+import { Organization } from './Organization'; // eslint-disable-line import/no-cycle
+import {
+  Address,
+  Category,
+  Eligibility,
+  Note,
+  Program,
+  Schedule,
+} from './Meta';
+
+// A Service is provided by an Organization
+export interface Service {
+  id: number
+  name: string
+  addresses?: Address[]
+  alsoNamed: string
+  alternate_name?: string|null
+  application_process: any
```

Comment:
Easy enough to find, was planning to evolve these as I saw them actually used in code moved to typescript, but you're right naive to assume only I would be involved in using them.

Updated all except RecurringSchedule which is a class not yet migrated to TS and a little more complex.

Reply:
Sounds good, thanks, and I can probably take care of RecurringSchedule, since I'm the one who was responsible for writing that in the the first place.